research/conditional/train/configs/baselines/gpt_medium.yaml

```diff
@@ -0,0 +1,40 @@
+# grid args
+runner: "research.conditional.train.cc_train"
+time: "0-02:00:00"
+n_gpus: 1
+interactive_debug: false
+
+# train params
+params:
+  # model
+  model_type: "gpt"
+  dataset_type: "c4"
+  dmodel: 512
+  dff: 2048
+  n_att_heads: 8
+  n_blocks: 8
+  cutoff: 128
```

Comment:
cutoff should be 256.
I've just became a fan of @Simontwice 's inheritance actually. But let's merge it without it.

Reply:
But target inheritance would define the whole dataset (and maybe even training steps, grad_clip, etc.) in the outermost file. Then dense baselines would inherit from those, adding their own model sizes etc. Then our ablations/custom models would inherit from dense baselines.

Reply:
Yeah that's what Ive been thinking as well.